package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "island.is",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
Maybe we should be updating this field according to the version number of the releases?

Reply:
As per the change in `nx.json`, this can be treated as when we want to trigger builds for all applications.
If that's the purpose, then it would clash with the actual release version number.
So IMO I don't see the benefit of keeping it in sync, but I see the benefit of keeping it separate. But it would be interesting to see the counter-argument?

Reply:
I'm not sure about this change here, why are we just first changing this version now? 
I'm wondering if we have to worry about any side effects from this. 
I think this is something @pshomov should take a look at.

Reply:
Since we're not using the package.json version for anything, we haven't had the need to change it before.
Can't think of any possible side effects, other than what this PR introduces, i.e. making projects affected by nx due to the change to nx.json.

Reply:
> As per the change in `nx.json`, this can be treated as when we want to trigger builds for all applications. If that's the purpose, then it would clash with the actual release version number. So IMO I don't see the benefit of keeping it in sync, but I see the benefit of keeping it separate. But it would be interesting to see the counter-argument?

Yes I don't have any counter argument other than just to be in sync, which is probably just the effort with no extra value.
So you changed this value to cause `package.json` to be affected and trigger build for everything? I can see the value in having that possibility, so maybe we just need to document it somewhere.

Reply:
True.
Yeah, needs to be documented 👍